internal/action/handle.go

```diff
@@ -14,19 +14,23 @@ import (
 )
 
 func getCurrent() string {
-	return registry.ReadNpm()
+	cur, err := registry.ReadNpm()
+	if (err) != nil {
```

Comment:
(err) 
typo ?

Reply:
May a typo. mistake. I will remove brackets

Reply:
Can view the latest commit. I fixed.